app/Census/CensusColumnRelationToHead.php

```diff
@@ -34,7 +35,7 @@ public function generate(Individual $individual, Individual $head = null) {
 		if ($head === null) {
 			return '';
 		} elseif ($individual == $head) {
-			return 'head';
+			return I18N::translate('Head of household');
```

Comment:
Ok. I will create a separate file for the german censuses (of Mecklenburg-Schwerin). 

I was only surprised that you use a hard coded word here. In the german censuses I know they use head of household most time instead of head only.